### ISSUE TITLE
Missing screen reader acesibility

### ISSUE THREAD

**Issue by okevinalmeida** [vlang/ui#182]
The only thing missing for this ui toolkit is screen reader accessibility. V itself is amasing, and this toolkit as well, but we blind programmers can't see anything on the app screen, our screen readers don't report anything

**Comment by medvednikov** [comment]
@okevinalmeida rest assured that V UI will have top notch accessibility. This is one of the main goals, I'll make it more clear.

This has been requested a lot.

**Comment by medvednikov** [comment]
I've added it to the 0.1 roadmap

https://github.com/vlang/ui/issues/31

**Comment by okevinalmeida** [comment]
Ok, thank you. I will be closing this and will be waiting to try things out. If you need any help with testing i'm available.

**Comment by medvednikov** [comment]
Let's keep it open until it's implemented.

**Comment by trypsynth** [comment]
Any updates on this issue?
Perhaps [AccessKit](https://accesskit.dev) would be of some use here?
